build.gradle

```diff
@@ -29,6 +29,7 @@ subprojects {
     targetCompatibility = JavaVersion.VERSION_1_8
     checkstyle {
         toolVersion = CheckStyleVersion
+        configDir = file("$rootProject.projectDir/quality-checks/checkstyle")
```

Comment:
The checkstyle plugin needs this config to build against JDK11 due to tighter security restrictions imposed by Java 11.